src/pages/wishListView/index.js

```diff
@@ -52,13 +117,68 @@ const WishListView = (props) => {
-          return (
-            <ListItem key={index} className={classes.li}>
-              <WishlistCard props={props} />
-            </ListItem>
-          );
-        })}
+        {list.length === 0 ? (
+          <DisplayMessage />
+        ) : (
+          list.map((item) => {
+            return (
+              <ListItem key={item.index} className={classes.li}>
+                <Container className={classes.root}>
+                  <Card props={props} onDelete={onDelete}>
+                    <CardHeader
+                      style={{
+                        backgroundImage: "url(/assets/wishlist/2.png)",
+                        backgroundSize: "cover",
+                      }}
+                      title={
+                        <Typography variant="h5" className={classes.titlestyle}>
+                          Fullstack Web Development Course <br /> by Code For Cause
+                        </Typography>
+                      }
+                      subheader={
+                        <Typography variant="subtitle2" className={classes.subheaderstyle}>
+                          90hrs of learning with Anuj Garg
+                        </Typography>
+                      }
+                      action={
+                        <IconButton className={classes.icon}>
+                          <DeleteOutline
+                            className={classes.delButton}
+                            index={item.index}
+                            onClick={onDelete}
+                          />
+                        </IconButton>
+                      }
+                      className={classes.cardHeader}
+                      classes={{
+                        title: classes.title1,
+                        subheader: classes.subheader,
+                      }}
+                    />
+                    <CardContent>
+                      <Typography variant="subtitle2">
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam
+                      </Typography>
+                      <Typography variant="caption" className={classes.captionset}>
+                        Classes Starting{" "}
+                        <span style={{ color: "#a60000" }}>16th March Onwards</span>
+                      </Typography>
+                      <Typography variant="body2" color="secondary">
+                        ₹ 94
+                      </Typography>
+                      <ButtonComponent
+                        className={classes.button}
+                        title="Checkout Now"
+                        onClick={() => history.push(`/course/${id}`)}
+                      />
+                    </CardContent>
+                  </Card>
+                </Container>
+              </ListItem>
+            );
+          })
+        )}
```

Comment:
WishlistCard is already created for that, it's redundant code. Please try to replace it with WishlistCard and pass props if needed for deletion feature